README.md

```diff
@@ -1,22 +1,26 @@
 # Chipy.org
 
-The code for the Chipy.org website
-This project is open source and the license can be found in LICENSE.
+This is the code for the Chipy.org website.
+This project is open source, and the license can be found in LICENSE.
 
 [![CircleCI](https://circleci.com/gh/chicagopython/chipy.org/tree/master.svg?style=svg)](https://circleci.com/gh/chicagopython/chipy.org/tree/master)
 
-Chipy.org is setup using the [12factor](http://12factor.net) methodology. The site is
+Chipy.org is set up using the [12factor](http://12factor.net) methodology. The site is
 normally powered by Heroku, but you can use Docker and Docker Compose for
 local development.
 
 ## Installation
 
-To get setup with chipy.org code it is recommended that you use the following:
+To get setup with chipy.org code, it is recommended that you use the following:
 
 * Docker - https://docs.docker.com/install/
 * docker-compose - https://docs.docker.com/compose/install/
 * make - https://www.gnu.org/software/make/
 
+For Windows users not using Windows 10 Pro, we recommend using Docker Toolbox from https://docs.docker.com/toolbox/toolbox_install_windows/ . After following the README instructions below, select the `default` virtual machine in VirtualBox. Set up port forwarding from the Guest Port as 8000 to the Host Port as 8000. Go to `localhost:8000` on your browser to see the site.
```

Comment:
Sure, I can implement the changes. Would it be alright with you if instead of saying `Set up port forwarding from the Guest Port as 8000 to the Host Port as 8000.`,  I put `Set up NAT port forwarding, where the Guest Port is 8000 and the Host Port is 8000. Guest IP and Host IP are left empty.` Either case is fine with me, but I thought the 2nd version would be more clear.

Reply:
The second version is good.  You can go with that.